content/tutorial-air-quality-analysis.md

```diff
@@ -0,0 +1,360 @@
+from May 31, 2019 to June 30, 2020. It has measurements of the standard pollutants that are required for Air Quality Index calculation and a few other important ones:
+Particulate Matter (PM 2.5 and PM 10), nitrogen dioxide (NO2), ammonia (NH3), sulfur dioxide (SO2), carbon monoxide (CO), ozone (O3), oxides of nitrogen (NOx), nitric oxide (NO), benzene, toluene, and xylene.
+
+Let's print out the first few rows to have a glimpse of our dataset.
+
+```{code-cell} ipython3
+! head air-quality-data.csv
+```
+
+For the purpose of this tutorial, we are only concerned with standard pollutants required for calculating the AQI, viz., PM 2.5, PM 10, NO2, NH3, SO2, CO, and O3. So, we will only import these particular columns with [np.genfromtxt](https://numpy.org/devdocs/reference/generated/numpy.genfromtxt.html). We'll initially create two sets: `pollutants_A` with PM 2.5, PM 10, NO2, NH3, and SO2, and `pollutants_B` with CO and O3. The
+two sets will be processed slightly differently, as we'll see later on.
+
+```{code-cell} ipython3
+pollutants_A = np.genfromtxt("air-quality-data.csv", dtype='f8', delimiter=",",
+                             skip_header=1, usecols=(1, 2, 3, 4, 5))
+pollutants_B = np.genfromtxt("air-quality-data.csv", dtype='f8', delimiter=",",
+                             skip_header=1, usecols=(6, 7))
+
+print(pollutants_A.shape)
+print(pollutants_B.shape)
+```
+
+Our dataset might contain missing values, denoted by `NaN`, so let's do a quick check with [np.isnan](https://numpy.org/devdocs/reference/generated/numpy.isnan.html).
+
+```{code-cell} ipython3
+print(np.where(np.isnan(pollutants_A) == True))
+print(np.where(np.isnan(pollutants_B) == True))
+```
+
+With this, we have successfully imported the data and checked that it is complete. Let's move on to the AQI calculations!
+
++++
+
+## Calculating the Air Quality Index
+
+
+We will calculate the AQI using [the method](https://app.cpcbccr.com/ccr_docs/FINAL-REPORT_AQI_.pdf) adopted by the [Central Pollution Control Board](https://www.cpcb.nic.in/national-air-quality-index) of India.  To summarize the steps:
+
+- Collect 24-hourly average concentration values for the standard pollutants; 8-hourly in case of CO and O3.
+
+
+- Calculate the sub-indices for these pollutants with the formula: 
+$$Ip = \dfrac{\text{IHi – ILo}}{\text{BPHi – BPLo}}\cdot{\text{Cp – BPLo}} + \text{ILo}$$
+
+    Where,
+
+    `Ip` = sub-index of pollutant `p`\
+    `Cp` = averaged concentration of pollutant `p`\
+    `BPHi` = concentration breakpoint i.e. greater than or equal to `Cp`\
+    `BPLo` = concentration breakpoint i.e. less than or equal to `Cp`\
+    `IHi` = AQI value corresponding to `BPHi`\
+    `ILo` = AQI value corresponding to `BPLo`
```

Comment:
Indenting by 4 spaces here moves this section into literal block. I'm not sure if this was intentional, but in case it wasn't it can be prevented by reducing the indentation level, e.g.
```suggestion
  Where,

  `Ip` = sub-index of pollutant `p`\
  `Cp` = averaged concentration of pollutant `p`\
  `BPHi` = concentration breakpoint i.e. greater than or equal to `Cp`\
  `BPLo` = concentration breakpoint i.e. less than or equal to `Cp`\
  `IHi` = AQI value corresponding to `BPHi`\
  `ILo` = AQI value corresponding to `BPLo`
```

Reply:
The indentation is on purpose so that everything is contained inside the bullet point.